usr/share/rear/lib/nbkdc-functions.sh

```diff
@@ -0,0 +1,54 @@
+# locate NovaBACKUP DC client configuration, referenced by
+#
+# prep/NBKDC/default/40_prep_nbkdc.sh
+# skel/NBKDC/etc/scripts/system-setup.d/59-start-nbkdc-client.sh
+
+
+# Set the NovaBACKUP DC default directory
+NBKDC_DIR=/opt/NovaStor/DataCenter
+
+
+function get_nbkdc_dir() {
+
+# check for running process rcmd-executor to determine its parent path
+NBKDC_DIR=$(ps -ef | grep rcmd-executor | grep -v grep | awk -F\  '{print $8}' | sed 's/\/rcmd-executor*//g')
```

Comment:
Why do you need this coupling with running daemons? Is this software always installed into a different path? Don't you  have some configs in `/etc` where you can read out the location?

Maybe `pgrep` can save you some complexity from this statement.